src/lib/config.ts

```diff
@@ -172,7 +173,7 @@ interface BasicConfig {
 	 *
 	 * @see https://marked.js.org/using_pro#extensions
 	 */
-	marked_extensions: marked.MarkedExtension[];
+	marked_extensions: marked.MarkedExtension;
```

Comment:
Hm I'm not really happy with this not being an array anymore however I also find the Marked extension API a bit confusing 😕 

Looking at https://marked.js.org/using_pro#use and the source of `@types/marked` again, and the way it's exposed by Marked, the most correct would be

```suggestion
	marked_extensions: marked.MarkedExtension[];
```

However I've looked at a few other published extension and they usually don't export a `MarkedExtension` but a `TokenizerExtension` or `RendererExtension`, and that's also how it's stated in the [custom extensions](https://marked.js.org/using_pro#extensions) docs.

Thus I think it's better to go with the first idea I had:

```suggestion
	marked_extensions: marked.MarkedExtension['extensions'];
```

and later do

```ts
marked.use({ extensions })
```

I'll open a bug ticket about this in Marked cause there's a discrepancy about this in their docs.

Reply:
I think it is good now :+1: Thank for your help